src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/form.tsx

```diff
@@ -259,7 +259,12 @@ class Form<P extends Props = Props, S extends State = State> extends React.Compo
             />
           )}
           <NumberField
-            label={t('Sampling Rate')}
+            label={
+              <React.Fragment>
+                {t('Sampling Rate')}
+                {' \u0025'}
```

Comment:
Any reason to not use '%' on its own?

Reply:
no, I just wanted to be consistent with other punctuation marks that we have in our codebase